nvme.c

```diff
@@ -5317,6 +5321,330 @@ static int show_registers(int argc, char **argv, struct command *cmd, struct plu
 	return err;
 }
 
+bool nvme_is_fabrics_reg(int offset)
+{
+	switch (offset) {
+	case NVME_REG_CAP:
+		fallthrough;
+	case NVME_REG_VS:
+		fallthrough;
+	case NVME_REG_CC:
+		fallthrough;
+	case NVME_REG_CSTS:
+		fallthrough;
+	case NVME_REG_NSSR:
+		fallthrough;
+	case NVME_REG_CRTO:
+		return true;
+	default:
+		break;
+	}
+
+	return false;
+}
+
+static int get_register_property(int fd, void **pbar, int offset)
```

Comment:
Looks like this is only for fabrics? "register" and "bar" are PCI terms, so I probably wouldn't use them here.

Reply:
This function calls the function nvme_get_property() only for fabric but this function itself for both PCI and fabrics. Also the register changed to property by the NVMe base specification revision 2.0c so seems okay to use both "register" and "bar".

Reply:
Also existing the function nvme_get_properties() calling the function nvme_get_property() uses "reg", "REG" and "bar".

Reply:
Fair enough